css/mistica-common.css

```diff
@@ -133,3 +249,331 @@
+}
+
+.mistica-tag-warning {
+    background: var(--vcolor-tagBackgroundWarning);
+    color: var(--vcolor-tagTextWarning);
+}
+
+.mistica-tag-error {
+    background: var(--vcolor-tagBackgroundError);
+    color: var(--vcolor-tagTextError);
+}
+
+/* Buttons */
+[class^='mistica-button'] {
+    display: inline-block;
+    border-radius: var(--mistica-border-radius-button);
+    font-size: var(--mistica-font-size-2);
+    line-height: var(--mistica-line-height-2);
+    font-weight: var(--mistica-font-weight-button);
+    border: 1.5px solid transparent;
+    box-sizing: border-box;
+    min-width: 104px;
+    padding: 12px 16px;
+    cursor: pointer;
+    /* Reset default browser styles */
+    outline: none;
+    font-family: inherit;
+    appearance: none;
+    user-select: none;
+    text-align: center;
+    text-decoration: none;
+    display: inline-block;
+    font-size: inherit;
+    --webkit-tap-highlight-color: transparent;
+}
+
+[class^='mistica-button']:hover,
+[class^='mistica-button']:active {
+    text-decoration: none;
+}
+
+[class^='mistica-button'][disabled] {
+    cursor: default;
+}
+
+[class^='mistica-button-primary'] {
+    color: var(--vcolor-textButtonPrimary);
+    background: var(--vcolor-buttonPrimaryBackground);
+}
+
+[class^='mistica-button-primary']:hover {
+    background: var(--vcolor-buttonPrimaryBackgroundHover);
+}
+
+[class^='mistica-button-primary']:active {
+    background: var(--vcolor-buttonPrimaryBackgroundPressed);
+}
+
+[class^='mistica-button-secondary'] {
+    color: var(--vcolor-textButtonSecondary);
+    background: transparent;
+    border-color: var(--vcolor-buttonSecondaryBorder);
+}
+
+[class^='mistica-button-secondary']:hover {
+    background: var(--vcolor-buttonSecondaryBackgroundHover);
+}
+
+[class^='mistica-button-secondary']:active {
+    background: var(--vcolor-buttonSecondaryBackgroundPressed);
+}
+
+[class^='mistica-button-danger'] {
+    color: var(--vcolor-textButtonError);
+    background: var(--mistica-color-buttonDangerBackground);
+}
+
+[class^='mistica-button-danger']:hover {
+    background: var(--mistica-color-buttonDangerBackgroundHover);
+}
+
+[class^='mistica-button-danger']:active {
+    background: var(--mistica-color-buttonDangerBackgroundPressed);
+}
+
+[class^='mistica-button-link'] {
+    color: var(--vcolor-textLink);
+    background: transparent;
+}
+
+[class^='mistica-button-link']:hover {
+    background: var(--vcolor-buttonLinkBackgroundPressed);
+}
+
+[class^='mistica-button-link']:active {
+    background: var(--vcolor-buttonLinkBackgroundPressed);
+}
+
+[class^='mistica-button'][class$='small'] {
+    padding: 6px 12px;
+    min-width: 80px;
+}
+
+/* Cards */
+.mistica-card,
+.mistica-snap-card,
+.mistica-display-card {
+    position: relative;
+    border: var(--mistica-boxed-border);
+    border-radius: var(--mistica-border-radius-container);
+    background: var(--mistica-color-backgroundContainer);
+    padding: 24px 16px;
```

Comment:
horizontal padding change between mobile and desktop

mobile
![image](https://github.com/user-attachments/assets/f30f0adb-40c1-4540-823e-e01a82976c9f)
desktop
![image](https://github.com/user-attachments/assets/7676bdc7-e092-4eb9-b3c6-a1cbefc4b5cc)

Reply:
changed